webui/react/src/components/ResourcePoolCard.tsx

```diff
@@ -88,6 +94,9 @@ const ResourcePoolCard: React.FC<Props> = ({ containerStates, resourcePool: rp }
     [],
   );
 
+  const descriptionClasses = [ css.description ];
+  if (!description) descriptionClasses.push(css.empty);
+
```

Comment:
**nit:** I've been mostly having these class definitions near the top of the component and hoisting all the variable definitions near the top of the function.